test/Dialect/FIRRTL/annotations.mlir

```diff
@@ -730,6 +730,19 @@ firrtl.circuit "Test" attributes {rawAnnotations = [
 
 // -----
 
+firrtl.circuit "Test" attributes {rawAnnotations =[
+  {class = "firrtl.linkage.Convention", target = "~Test|Test", convention = "scalarized"},
+  {class = "firrtl.linkage.Convention", target = "~Test|Test2", convention = "internal"}
+  ]} {
+  // CHECK: attributes {convention = #firrtl<convention scalarized>}
+  firrtl.module @Test() attributes {convention = #firrtl<convention internal>} {}
+
+    // CHECK-NOT: attributes {convention = #firrtl<convention scalarized>}
+  firrtl.module @Test2() attributes {convention = #firrtl<convention scalarized>} {}
```

Comment:
Nit: leading whitespace. Possible an affirmative check of `internal` is better?

Reply:
Good catch on whitespace. The convention isn't printed when it's internal, so there's nothing to match against.